src/Icons/src/Svg/SvgUtils.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Symfony\UX\Icons\Svg;
+
+final class SvgUtils
+{
+    /**
+     * Transforms a valid icon ID into an icon name.
+     *
+     * @throws \InvalidArgumentException if the ID is not valid
+     * @see isValidId()
+     */
+    public static function idToName(string $id): string
```

Comment:
Could these be added to the Icon object in https://github.com/kbond/ux/pull/7?

Reply:
If so, then combine this PR with an updated #7.

Reply:
Will do in a moment !

Reply:
https://github.com/kbond/ux/pull/9